### ISSUE TITLE
demo for a single image

### ISSUE THREAD

**Issue by hosnasattar** [mks0601/PoseFix_RELEASE#39]
could you provide a demo running on a single image?

**Comment by snitchjinx** [comment]
Hello! I wonder if it's feasible to test the pre-trained COCO-based posefix on a name_of_input_pose.json outside the COCO dataset. Is the result in name_of_input_pose.json confined to image samples from COCO dataset? Thanks!

**Comment by mks0601** [comment]
Sorry for late reply. 
@hosnasattar For the single image input, you only need to hack `dataset.py`. For example, you can edit `data/COCO/dataset.py` to load a single image and you can run the same command described in the README. Could you try this trick?

@snitchjinx Like the above answer, you can hack `dataset.py`. The pre-trained PoseFix model is not restricted to the images in the COCO dataset. You can try any images with the initial pose.

**Comment by hosnasattar** [comment]
@mks0601 I can give it a try but I think it would be great in general if you could provide a demo for a single image.

**Comment by mks0601** [comment]
Yeah I agree but I'm busy doing other projects :( I'll try when I have time. Thanks!
